Core/src/Scene.cpp

```diff
@@ -33,23 +33,16 @@ void Scene::SpawnSceneDefinition()
     lightEntity->transform->setLocalOrientation(glm::quat(-0.5f, 0.5f, 0.5f, 0.f));
     EntityManager::getInstance().Instantiate(lightEntity);
 
-    //(glm::vec3(0.f, 5.f, 0.f), glm::vec3(120.0f, -40.0f, 0.0f), LightType::DirectionalLight);
-
     // TODO: Refactor to a Single Car Entity with Mesh and Rigidbody components
+    // 
+    // TODO: Figure out location of assets/non code files within solution

```

Comment:
Unreal and unity typically consider a specific folder as root for assets and then has a package path that tells an asset loader how to traverse the asset folder to get to that specific thing. They also have specifically an asset definition that translates a generic uasset or asset file into a variety of other types but I dont think we have the time or luxury of doing that so maybe hard linking things should be fine for our use cases.

Reply:
Currently we have an Asset folder inside Core. I don't believe this is the best location for it however. Discussion on where exactly to place this folder should be had, also applies to our Shader folder in Renderer.